.circleci/config.yml

```diff
@@ -30,10 +43,27 @@ jobs:
     executor: python37
     steps:
       - run_tests
+  test38:
+    executor: python38
+    steps:
+      - run_tests
+  pypi_deploy:
+    executor: python37
+    steps:
+      - deploy_to_pypi
 
 workflows:
   version: 2
   test:
     jobs:
       - test36
       - test37
+      - test38
+  deploy:
+    jobs:
+      - pypi_deploy:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
I think that .* only tells that tag should have its parts separated by periods- So yes this is a good idea and I added this as you suggested.

Reply:
No, `/.*/` is a regular expression that matches anything, which is bad. `.` in a regex means "any character" and the `*` means "zero or more repetitions of what is in front of me".

This might come in handy: https://regex101.com/r/ZiTlcd/1

Reply:
I might have forgotten a meaning of this regex so thanks a lot for that.